### PR TITLE
Adding image file for portfolio.

### PR DESCRIPTION
**Pull request by frostyclawz**
Image to add to portfolio docs. Will add more images over some time. Also, a minor change to README.md

### CODE REVIEW

File: portfolio/src/main/webapp/style.css [hunk 1]
```diff
@@ -7,3 +7,16 @@
 #greeting-container {
   margin-top: 20px;
 }
+
```
Comment (ianjjohnson) [inline portfolio/src/main/webapp/style.css:10]: nit: extra whitespace here (can remove one blank line)

File: portfolio/src/main/webapp/style.css [hunk 2]
```diff
@@ -7,3 +7,16 @@
...
+
+.center {
```
Comment (ianjjohnson) [inline portfolio/src/main/webapp/style.css:12]: Is this selector being used for anything?

Comment (frostyclawz) [reply portfolio/src/main/webapp/style.css:12]: No, it's just extra whitespace I made on accident when working with the CSS.

Comment (ianjjohnson) [reply portfolio/src/main/webapp/style.css:12]: If the .center selector isn't being used, can you delete it from the CSS file?
(If it is being used, can you delete the extra blank line above it?)

**Pull request merged**